box.json.dist

```diff
@@ -1,23 +1,16 @@
 {
-    "alias": "psalm.phar",
```

Comment:
there shouldn't be a need for an alias unless you really want to refer to `phar://psalm.phar` in the codebase which I didn't see

Reply:
Yeah, this was just added for renaming in https://github.com/vimeo/psalm/issues/595, but you fixed later.